test/annotation/utils_tests.cxx

```diff
@@ -0,0 +1,82 @@
+#define BOOST_TEST_MODULE
```

Comment:
Should define a test suite name? See https://www.boost.org/doc/libs/1_68_0/libs/test/doc/html/boost_test/tests_organization/test_suite/master_test_suite.html#ref_BOOST_TEST_MODULE